Robus/src/robus.c

```diff
@@ -244,6 +244,7 @@ static error_return_t Robus_ResetNetworkDetection(ll_container_t *ll_container)
     do
     {
         Robus_SendMsg(ll_container, &msg);
+        while(MsgAlloc_TxAllComplete() != SUCCEED);
```

Comment:
Please add some more details on the commit comment defining what your are doing here.